### DISCUSSION TITLE
Unread thread indicator for forum module, and ability to view all unread content

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [NamelessMC/Nameless#2166] [Ideas]
**Please describe the feature in as much detail as possible**
Some symbol just after forum icon, which can for example glow maybe, to indicate that there are new posts/threads that you as site/forum user didn't read.

**Comment by tadhgboyle** [comment]
This is a great idea actually! I'm working on a few things for Nameless right now, but when I have some time I'll take a look at doing this too, thanks for the suggestion
